data/data/bootstrap/files/etc/profile.d/proxy.sh.template

```diff
@@ -0,0 +1,5 @@
+{{if .Proxy}}
```

Comment:
I don't know that we care, but this will result in empty lines in the file.

To do it without empty lines,
```
{{ if .Proxy -}}
{{ if .Proxy.HTTPProxy -}}
export HTTP_PROXY="{{ .Proxy.HTTPProxy }}"
{{ end -}}
...
{{ end -}}
```